package.json

```diff
@@ -78,7 +78,7 @@
   },
   "lint-staged": {
     "*.ts": [
-      "npm run lint-fix"
+      "echo 'Lint-staged deshabilitado temporalmente'"
```

Comment:
why this?

Reply:
"This is necessary because Husky automatically fixes prettier errors before making the commit. Here, I'm disabling it so that the prettier error passes through and gets detected by the GitHub Action.